deploy/services-demo/create_test_team_members.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+set -ex
+
+ADMIN_UUID="a09e9521-e14e-4285-ad71-47caa97f4a16"
+TEAM_UUID="9e57a378-0dca-468f-9661-7872f5f1c910"
+BRIG_HOST="http://localhost:8082"
+START="1"
+COUNT="1"
+CSV="false"
+
+USAGE="
+This bash script can be used to create active members in a given team.
+
+Note that this uses internal brig endpoints.  It is not exposed over
+nginz and can only be used if you have direct access to brig.
+
+USAGE: $0
+    -a <admin uuid>: User ID of the inviting admin.  default: ${ADMIN_UUID}
+    -t <team uuid>: ID of the inviting team.  default: ${TEAM_UUID}
+    -s <S>: Start at offset. default: ${START}
+    -n <N>: Create <N> users. default: ${COUNT}
+    -h <host>: Base URI of brig. default: ${BRIG_HOST}
+    -c: Output as headerless CSV in format 'User-Id,Email,Password'. default: ${CSV}
+"
+
+# Option parsing:
+# https://sookocheff.com/post/bash/parsing-bash-script-arguments-with-shopts/
+while getopts ":a:t:s:n:h:c" opt; do
+  case ${opt} in
+    a ) ADMIN_UUID="$OPTARG"
+      ;;
+    t ) TEAM_UUID="$OPTARG"
+      ;;
+    s ) START="$OPTARG"
+      ;;
+    n ) COUNT="$OPTARG"
+      ;;
+    h ) BRIG_HOST="$OPTARG"
+      ;;
+    c ) CSV="true"
+      ;;
+    : ) echo "-$OPTARG" requires an argument 1>&2
+        exit 1
+      ;;
+    \? ) echo "$USAGE" 1>&2
+         exit 1
+      ;;
+  esac
+done
+shift $((OPTIND -1))
+
+if [ "$#" -ne 0 ]; then
+  echo "$USAGE" 1>&2
+  exit 1
+fi
+
+# Generate users
+END=$((COUNT + START - 1))
+for i in $(seq "$START" "$END")
+do
+    EMAIL='w'$(printf "%03d" "$i")"@example.com"
```

Comment:
perhaps a comment or something to tell the user the emails start with 'w'?

Reply:
One thing that we should be careful about is that we _actually_ send out emails with this request.
Sending emails to _example.com_ will trigger an email being sent out and may lower our reputation.
Perhaps make the domain configurable and explain that, currently, we _effectively_ send out the emails (and perhaps we need an extra endpoint/flag that prevents sending it over the Wire?)
@fisx

Reply:
Sorry, only saw that after I merged...  I'll make another PR and ping you.